src/app/settings/components/project-role-edit-panel/project-role-edit-panel.tsx

```diff
@@ -20,6 +22,10 @@ interface ProjectRoleEditPanelProps {
 
 export class ProjectRoleEditPanel extends React.Component<ProjectRoleEditPanelProps, any> {
 
+    public static contextTypes = {
```

Comment:
Looks like context is not used anywhere. Please remove contextTypes field

Reply:
Once context is removed you might convert component to stateless component:

```
export const ProjectRoleEditPanel = (props: ProjectRoleEditPanelProps) => (
...
);
```

Reply:
Ok removed the context and converted it into a stateless component